account_payment_return/models/payment_return_reason.py

```diff
@@ -20,3 +20,9 @@ def name_search(self, name, args=None, operator='ilike', limit=100):
         if not recs:
             recs = self.search([('name', operator, name)] + args, limit=limit)
         return recs.name_get()
+
+    @api.multi
+    @api.depends('code', 'name')
```

Comment:
This is not needed, as it's not a compute field

Reply:
Right, I removed it.